### ISSUE TITLE
Release 2.0.0

### ISSUE THREAD

**Issue by grant** [cloudevents/sdk-java#172]
Hello team,
Do we know when a `2.0.0` release will be cut?

Looks like @slinkydeveloper said there is some work to do.
https://github.com/cloudevents/sdk-java/issues/33#issuecomment-642620220
Is there a list of issues needed to be fixed before releasing?

The last cut was a few months ago:
https://search.maven.org/artifact/io.cloudevents/cloudevents-api

Diff: https://github.com/cloudevents/sdk-java/compare/v1.3.0...HEAD

An update, or reason/tracking issue would help me understand what we need to do before we can get a `2.0.0` out. Looking to support a different package: https://github.com/GoogleCloudPlatform/functions-framework-java/issues/37
Thanks!

**Comment by slinkydeveloper** [comment]
Hi Grant, there is a project explaning the issues todo/done https://github.com/cloudevents/sdk-java/projects/1

**Comment by grant** [comment]
Thanks for the project, didn't see that before.

**Comment by slinkydeveloper** [comment]
@grant can we close it since it's a duplicate of the GH project?

**Comment by grant** [comment]
@slinkydeveloper I mean... which issue can I follow to check the completion of the GH project? Is there a different issue tracking publishing to Maven Central? Usually a project and issues are separate.

**Comment by slinkydeveloper** [comment]
Maybe if you follow the repo you follow the project updates? Or maybe there is a button to follow the project updates? I don't know :man_shrugging:...
Anyway, I hope to do soon a first milestone release (hopefully next week) and i'll announce it on slack, so you won't lose it :wink:
